### ISSUE TITLE
Broken link to  ITCPTransport API doc in core clients howto

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4348]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)| @jesstess reported|
|-|-|
|Trac ID|trac#4348|
|Type|defect|
|Created|2010-03-04 21:11:53Z|
|Branch|https://github.com/twisted/twisted/tree/broken-client-link-4348|

<details><summary>Searchable metadata</summary>

```
trac-id__4348 4348
type__defect defect
reporter__jesstess jesstess
priority__normal normal
milestone__ 
branch__branches_broken_client_link_4348 branches/broken-client-link-4348
branch_author__jesstess jesstess
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1267737113000000 1267737113000000
changetime__1268773387000000 1268773387000000
version__None None
owner__ 
cc__jesstess cc__thijs cc__khorn
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_1"></a>|@jesstess commented|
|-|-|

(In [28658]) Branching to 'broken-client-link-4348'

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_2"></a>|@jesstess commented|
|-|-|

(In [28659]) Fix broken API link, remove trailing whitespace, spell check, add
markup, make logBot example code match the actual ircLogBot.py.

refs [#4348](https://github.com/twisted/twisted/issues/4348)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @jesstess|
|-|-|

There's a sentence that goes "...sets on it an attribute called <code>factory</code>". What about "...sets a <code>factory</code> attribute (on it)".

If you think that's an improvement as well, please fix and then merge it, thanks.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_6"></a>|@jesstess commented|
|-|-|

(In [28750]) Reword how a factory's protocol has a reference to the factory so it
can access persistent state, as that does seem to confuse people.

refs [#4348](https://github.com/twisted/twisted/issues/4348)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_7"></a>|@jesstess set status to `closed`|
|-|-|

(In [28752]) Merge broken-client-link-4348

Author: jesstess
Reviewer: thijs
Fixes: [#4348](https://github.com/twisted/twisted/issues/4348)

Fix broken API link, remove trailing whitespace, spell check, add
markup, make logBot example code match the actual ircLogBot.py.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
